lib/msf/core/option_container.rb

```diff
@@ -18,6 +18,7 @@ module Msf
   autoload :OptRaw, 'msf/core/opt_raw'
   autoload :OptRegexp, 'msf/core/opt_regexp'
   autoload :OptString, 'msf/core/opt_string'
+  autoload :OptRhostUrl, 'msf/core/opt_rhost_url'
```

Comment:
```suggestion
  autoload :OptRhostURL, 'msf/core/opt_rhost_url'
```

https://github.com/rubocop-hq/ruby-style-guide#camelcase-for-classes